lib/rubocop/cop/root_cops/eq_be_eql.rb

```diff
@@ -0,0 +1,30 @@
+module RootCops
+  class EqBeEql < ::RuboCop::Cop::Cop
+    MSG = "Prefer `eq` over `be` and `eql` when checking booleans, numbers, symbols, strings and nil".freeze
+
+    def_node_matcher :be_or_eql, <<-PATTERN
+      (send _ {:to :to_not :not_to} $(send nil? {:be :eql} {true false int float sym nil_type? str_type?}))
```

Comment:
Do we want to enforce this for everything or do we have valid use cases for `be` elsewhere?

Reply:
There are some valid cases for be. Examples:
 - `be >= #`
 - `be_present`
 - `be_in[]`

Reply:
Ah gotcha, thanks 🚀